scripts/umbrel-os/external-storage/monitor

```diff
@@ -38,6 +45,13 @@ main () {
       break
     fi
 
+    if [[ $(dmesg | grep --quiet "uas_abort_handler") > 0 ]]; then
+      echo "External storage might have UAS issues"
+      sed "s/usb-storage.quirks=/usb-storage.quirks=$(get_uas_string),/g" -i /boot/cmdline.txt
+      echo "Rebooting"
+      sudo reboot
```

Comment:
We need to be really careful here if we're gonna try to auto patch and reboot. If we mess something up we risk sending the user into a permanent boot loop.

Like if there's some other reason the `uas_abort_handler` string ends up in dmesg or if for whatever reason adding the device to `usb-storage.quirks` doesn't resolve the issue, the user will be stuck in a boot loop.

A simple sanity check to test if `get_uas_string` already exists in `usb-storage.quirks` would go a long way to preventing this. If it already exists just don't do anything and continue to boot.